database/controllers/index.js

```diff
@@ -4,36 +4,53 @@ module.exports = {
     dbm,
```

Comment:
you dont have to require the dbm here and then export it. You already exported the functions in dbm. Other files just need to include that. Unless I am missing something.

Reply:
No this is my inexperience coming through here as I followed some guides on how to write the methods and then call them.  If we don't need the index.js page at all then I don't mind removing it.

Reply:
Yeah pretty sure it can be removed.

Reply:
Removed this file completely.